### ISSUE TITLE
Wrong version of pycharm community version is installed

### ISSUE THREAD

**Issue by arsirantala** [microsoft/winget-pkgs#15318]
Pycharm community version installation uses old installation package.

Run winget upgrade --all

Have some version of pycharm community version installed.

Winget downloads https://download.jetbrains.com/python/pycharm-community-2020.3.3.exe when there is pycharm-community-2021.1.1.exe available from the official jetbrains site.

WHen pycharm is started it still shows the 2021.1 CE version, but there seems some other pycharm installations alongside with the latest version I had manually installed.

Windows 10 Pro 20H2

```none
[winget --info]
Windows Package Manager v1.0.11451
Windows: Windows.Desktop v10.0.19042.985
Package: Microsoft.DesktopAppInstaller v1.11.11451.0

Any other software?
```

**Comment by abdonmorales** [comment]
It seems that the manifest needs to be add or there has been a change in version because of JetBrains marketing version and build version

[https://github.com/microsoft/winget-pkgs/tree/master/manifests/j/JetBrains/PyCharm/Community](url)

**Comment by OfficialEsco** [comment]
I think this issue is a bit outdated/mid transfer from fix...
It got fixed 3 days ago https://github.com/microsoft/winget-pkgs/tree/master/manifests/j/JetBrains/PyCharm/Community

![image](https://user-images.githubusercontent.com/15158490/120504301-50b55700-c3c4-11eb-9776-de9eb3cef898.png)

![image](https://user-images.githubusercontent.com/15158490/120504389-61fe6380-c3c4-11eb-89f4-063a265b8c50.png)

**Comment by abdonmorales** [comment]
@OfficialEsco You can close the issue as it will be resolved once I add the build numbers to the PyCharm Community folder

**Comment by OfficialEsco** [comment]
I can't, you can add closes/resolves/fixes #15318 to your PR text to close the case which links and closes the case when the PR is merged
